dev-infra/format/format.ts

```diff
@@ -47,7 +51,7 @@ export async function checkFiles(files: string[]) {
     // Provide output expressing which files are failing formatting.
     info.group('\nThe following files are out of format:');
     for (const file of failures) {
```

Comment:
We need to expose the file path here as well.

```suggestion
    for (const {filePath} of failures) {
      info(`• ${filePath}`);
```

Reply:
oops. good catch.